backends/beats/winlogbeat/render.go

```diff
@@ -132,6 +136,11 @@ func (wlbc *WinLogBeatConfig) RenderOnChange(response graylog.ResponseCollectorC
 		}
 	}
 
+	// gl2_source_collector can't be set with Winlogbeat so we use the shipper name
```

Comment:
This seems to be an issue in our beats plugin: https://github.com/Graylog2/graylog-plugin-beats/blob/master/src/main/java/org/graylog/plugins/beats/BeatsCodec.java#L187